src/Avalonia.Themes.Default/TreeViewItem.xaml

```diff
@@ -48,15 +59,27 @@
   <Style Selector="TreeViewItem /template/ ContentPresenter#PART_HeaderPresenter">
     <Setter Property="Padding" Value="2"/>
   </Style>
-  
-  <Style Selector="TreeViewItem:selected /template/ ContentPresenter#PART_HeaderPresenter">
-    <Setter Property="Background" Value="#fff0f0f0"/>
+
+  <Style Selector="TreeViewItem[IsSelected=false] /template/ Border#SelectionBorder[IsPointerOver=true]">
```

Comment:
It's best to use pseudoclasses where possible for performance reasons. Changing the selectors to be as below saves 400kb on the TreeView page in ControlCatalog:

```
  <Style Selector="TreeViewItem /template/ Border#SelectionBorder[IsPointerOver=true]">
    <Setter Property="Background" Value="{DynamicResource ThemeControlHighlightMidBrush}"/>
  </Style>

  <Style Selector="TreeViewItem:selected /template/ Border#SelectionBorder">
    <Setter Property="Background" Value="{DynamicResource ThemeAccentBrush4}"/>
  </Style>

  ...etc
```

Reply:
I can do something like the following?

```
  <Style Selector="TreeViewItem /template/ Border#SelectionBorder:pointerover">
    <Setter Property="Background" Value="{DynamicResource ThemeControlHighlightMidBrush}"/>
  </Style>

  <Style Selector="TreeViewItem:selected /template/ Border#SelectionBorder">
    <Setter Property="Background" Value="{DynamicResource ThemeAccentBrush4}"/>
  </Style>

  <Style Selector="TreeViewItem:selected /template/ Border#SelectionBorder:focus">
    <Setter Property="Background" Value="{DynamicResource ThemeAccentBrush3}"/>
  </Style>

  <Style Selector="TreeViewItem:selected /template/ Border#SelectionBorder:pointerover">
    <Setter Property="Background" Value="{DynamicResource ThemeAccentBrush3}"/>
  </Style>

  <Style Selector="TreeViewItem:selected /template/ Border#SelectionBorder:pointerover:focus">
    <Setter Property="Background" Value="{DynamicResource ThemeAccentBrush2}"/>
  </Style>
```